server/src/main/java/org/elasticsearch/cluster/metadata/Template.java

```diff
@@ -213,7 +213,7 @@ public String toString() {
 
     @Override
     public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
-        return toXContent(builder, params, null, null);
+        return toXContent(builder, params, null, null, false);
```

Comment:
Same here about these always being non-system?

Reply:
Same as above, this is a template that is not yet applied to either system or non-system data streams. I _think_ that CompositeIndexTemplate is the only template-like place where we actually put things together and apply them to a data stream.